exampleSite/config.yml

```diff
@@ -62,6 +62,11 @@ params:
   readingTimeText: "Estimated reading time:"
   fathomUrl: ""
   fathomSiteId: ""
+  # If you want to use plausible(https://plausible.io) for analytics, add this section
+  plausibleAnalytics:
+    domain: "example.com"
```

Comment:
Please use empty strings as defaults, we dont want to enable analytics on example site.